internal/cmd/flink/command_iam_binding.go

```diff
@@ -0,0 +1,26 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+)
+
+type iamBindingOut struct {
+	Id           string `human:"ID" serialized:"id"`
+	Cloud        string `human:"Cloud" serialized:"cloud"`
+	Region       string `human:"Region" serialized:"region"`
+	Environment  string `human:"Environment" serialized:"environment"`
+	IdentityPool string `human:"Identity Pool" serialized:"identity_pool"`
+}
+
+func (c *command) newIamBindingCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "iam-binding",
+		Short: "Manage Flink IAM bindings.",
```

Comment:
Can you include a `Long` description that explains what IAM stands for?

Reply:
Added!